ci/Jenkinsfile_utils.groovy

```diff
@@ -18,14 +18,24 @@
 // under the License.
 
 // initialize source codes
-def init_git() {
+def init_git(git_sha = '') {
+
+  git_sha = git_sha ? git_sha.trim() : ''
```

Comment:
Git should always be manage by Jenkins. Doing a manual override of the git-state can cause quite a lot of nasty things.

Thus, if you'd like to work off a certain Git hash, invoke the job with the correct source.

Reply:
On second thought. I have removed the MXNET_SHA. I will leave it in the artifact repository though. I would like to have the affordance of overriding the commit_id that gets pulled from the artifact repository. This will be useful when developing (so you don't need compile mxnet every time when developing the pipeline for a particular release channel).

Reply:
Lets have a chat about that on Monday.

Reply:
Shouldn't be a blocker

Reply:
Unfortunately it is a blocker since that's messing with Jenkins internals

Reply:
I've removed it. So, it's no longer a blocker.

Reply:
I've removed it. So, it's no longer a blocker.